src/js/BookReader/Mode1Up.js

```diff
@@ -226,38 +304,39 @@ export class Mode1Up {
       // current index in drawLeafsOnePage after we create the new view container
 
       // Make sure this will count as current page after resize
-      const fudgeFactor = (this.book.getPageHeight(this.br.currentIndex()) / this.br.reduce) * 0.6;
+      const fudgeFactor = 0.6 * this.physicalInchesToDisplayPixels(this.book.getPage(this.br.currentIndex()).heightInches);
       const oldLeafTop = this.getPageTop(this.br.currentIndex()) + fudgeFactor;
-      const oldViewDimensions = this.calculateViewDimensions(this.br.reduce, this.br.padding);
+      const oldViewDimensions = this.calculateViewDimensions();
```

Comment:
so happy to see the step to move away from `br.reduce` as a main dimension to calculate book/image size against.

Reply:
Bye bye bye!